src/components/MapComponent.vue

```diff
@@ -1,5 +1,7 @@
 <template>
   <div class="map-container">
+    <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.4.0/mapbox-gl-draw.css" type="text/css">
```

Comment:
Do we want to use a CDN for these stylesheets, or would we rather host these ourselves?

Reply:
used mdi instead